zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterOption.java

```diff
@@ -21,7 +21,7 @@
  *
  */
 public class InterpreterOption {
-  boolean remote;
+  boolean remote = false;
```

Comment:
I think default value should be 'true' to allow previous version of zeppelin work with it.